src/result_object/index.js

```diff
@@ -64,7 +64,7 @@ class ResultObject {
      * @param {string} severity - Whether it is a `fail` or `warn` test.
      * @return {Function} Calling a helper function
      */
-    fail(fieldName: string, statement: string, severity: Severity) {
+    fail(fieldName: string, statement: string, severity: Severity = FAIL) {
         return fail.call(this, fieldName, statement, severity);
```

Comment:
The famous `this`

Reply:
Why to define the default severity here and not [here](https://github.com/fiverr/passable/blob/90b45126270af0b0e69dbe16a8c18102508b8ae2/src/Passable.js#L34) ?

Reply:
solved by itself :)